src/pages/Home.tsx

```diff
@@ -122,11 +135,14 @@ const Home = ({ setCartCount }: HomeProps) => {
       <p className="text-xl mb-6 text-text-secondary">
         Discover and collect your favorite vinyl records
       </p>
-
       <LatestReleases vinyl={latestReleases} addToCart={addToCart} />
       <StaffPicks vinyl={staffPicks} addToCart={addToCart} />
-      <GenreSection genres={genres} genreVinyls={genreVinyls} />
-
+      <GenreSection
+        genres={genres}
+        genreVinyls={genreVinyls}
+        coversByGenre={coversByGenre}
+      />{" "}
+      {/* Pass coversByGenre */}
       <section className="mb-12 max-w-90" aria-labelledby="newsletter">
         <h3
```

Comment:
I think this should be h2 because there is no h2 above it in this section (it is not a subheading of any h2 element that exists on the page, it is its own heading for the newsletter)